client/MajorComponents/ShippingComponents/MeetYourSeller.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function MeetYourSeller(props) {
+  return (
+    <div id="MeetYourSellerContainer">
+      {/* <div id="ListingSeller">
+        seller name: {props.ItemSeller.sellerName}
+      </div>
+      <div id="ListingSellerSales">
+        {props.totalSales} sales
+      </div>
+      <div id="ListingAverageRating">
+        average rating: {props.averageRating}
+      </div> */}
+    </div>
```

Comment:
does this function just render an empty div????

Reply:
definitely should be using styled components or css modules to create a styled div. There is no reason you would need a functional component to render a container